components/gitpod-db/src/typeorm/migration/1633334251143-IndexCodeSyncCreated.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import {MigrationInterface, QueryRunner} from "typeorm";
+import { indexExists } from "./helper/helper";
+
+const INDEX_NAME = "ind_dbsync";
+const TABLE_NAME = "d_b_code_sync_resource";
+
+export class IndexCodeSyncCreated1633334251143 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        if (!(await indexExists(queryRunner, TABLE_NAME, INDEX_NAME))) {
```

Comment:
FYI, today I learned about this helper (which doesn't seem to be used anywhere):

https://github.com/gitpod-io/gitpod/blob/af51739d341bb2245598e275336ae9f730e3b41a/components/gitpod-db/src/typeorm/migration/helper/helper.ts#L9-L18

However, no action needed here (feel free to resolve).